src/lib/tls/tls12/tls_record.cpp

```diff
@@ -89,13 +89,13 @@ std::vector<uint8_t> Connection_Cipher_State::aead_nonce(uint64_t seq, RandomNum
       case Nonce_Format::AEAD_XOR_12: {
          std::vector<uint8_t> nonce(12);
          store_be(seq, nonce.data() + 4);
-         xor_buf(nonce, m_nonce.data(), m_nonce.size());
+         xor_buf(std::span(nonce), std::span(m_nonce));
```

Comment:
```suggestion
         xor_buf(nonce, m_nonce);
```

... vectors are ranges by itself. No need for the spans.